metron-platform/metron-elasticsearch/metron-elasticsearch-common/pom.xml

```diff
@@ -26,7 +26,7 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <guava_version>${global_hbase_guava_version}</guava_version>
+        <guava_version>${global_guava_version}</guava_version>
```

Comment:
I will take a look at this more carefully and see if we need to alter the Guava version here.

Reply:
This is not needed.  Removed it.